napari_allencell_annotator/_tests/view/viewer_test.py

```diff
@@ -0,0 +1,113 @@
+from pathlib import Path
+
+import napari
+from bioio import BioImage
+
+import napari_allencell_annotator
+from napari.layers import Points
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+from napari_allencell_annotator.view.viewer import Viewer, PointsLayerMode
+import numpy as np
+import pytest
+from napari.utils import Colormap
+
+
+@pytest.fixture
+def viewer(make_napari_viewer: napari.Viewer) -> Viewer:
+    return Viewer(make_napari_viewer())
+
+
+def test_all_points_layer(viewer: Viewer) -> None:
+    # ARRANGE
+    viewer.viewer.add_shapes()
```

Comment:
why are we using an `add_shapes()` method here?

we either need to implement this in our viewer, or use a different method to ensure we're only grabbing points layers. I think adding an empty image should suffice.

Reply:
https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/39fdc696e96c353c6b3ae8f020ce73eacd2b55a6 I use a (2, 2, 2, 2) image to test since it says ValueError: Image data must have at least 2 dimensions when I use an empty image.